packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -354,16 +354,24 @@ export class Slider extends FormAssociated<HTMLInputElement>
     /**
      *  Handle mouse moves during a thumb drag operation
      */
-    private handleMouseMove = (e: MouseEvent): void => {
+    private handleMouseMove = (e: MouseEvent | TouchEvent): void => {
         if (this.readOnly || this.disabled || e.defaultPrevented) {
             return;
         }
 
-        // update the value based on current position
-        const eventValue: number =
-            this.orientation === Orientation.horizontal
-                ? e.pageX - this.trackLeft
-                : e.pageY;
+        let eventValue: number = 0;
+
+        if (e instanceof MouseEvent) {
+            eventValue =
+                this.orientation === Orientation.horizontal
```

Comment:
Seems like you could set an object to be the either e or e.touches[0] and then use the same logic to get the calculated value, would be a little more compact and only 1 place for the calc

Reply:
I considered doing that as a source event or possibly a getPosition method but I wasn't sure how to type it. I'll play around with it.

Reply:
Created a source event that points to e.touches when it's a touch event, otherwise it uses the event.

Reply:
Yes! love it, just smoking again and will approve